client.cpp

```diff
@@ -99,7 +103,7 @@ class VoltCallback : public voltdb::ProcedureCallback {
         zend_list_addref(m_res_id);
     }
 
-    bool callback(voltdb::InvocationResponse response) {
+    bool callback(voltdb::InvocationResponse response) throw (voltdb::Exception){
```

Comment:
I don't think you can throw C++ exceptions in a PHP extension. They won't be handled. You have to catch all exceptions thrown by our C++ client and turn them into proper PHP exceptions or error return codes.